src/dymaptic.GeoBlazor.Core/Serialization/EnumToKebabCaseStringConverter.cs

```diff
@@ -68,4 +69,29 @@ public override T Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerial
             return default(T)!;
         }
     }
+}
+
+/// <summary>
+///     Converts an enum to a kebab case string for serialization. Used with LabelPlacement which returns esriServerPointLabelPlacement from the ESRI JS.
+/// </summary>
+public class LabelPlacementStringConverter : EnumToKebabCaseStringConverter<LabelPlacement>
+{
+    /// <inheritdoc />
+    public override LabelPlacement Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        string? value = reader.GetString()
+            ?.Replace("-", string.Empty)
+            .Replace("esriServerPointLabelPlacement", string.Empty);
+
+        try
+        {
+            return value is not null ? (LabelPlacement)Enum.Parse(typeof(LabelPlacement), value, true) : default(LabelPlacement)!;
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine(ex);
```

Comment:
should this be a console log?

Reply:
In general, I've never hooked up GeoBlazor to ILogger, although I suppose we could/should. Since it's a library, I wasn't sure how I felt about "pushing" log messages to someone's logger implementation? Anyways, that seems like it should be it's own library-wide gissue. Let's talk about it more.